test/corpus/expressions.txt

```diff
@@ -1167,63 +1088,53 @@ void m() {
             type: (implicit_type)
             (variable_declarator
               name: (identifier)
-              (equals_value_clause
-                (anonymous_method_expression
-                  (modifier)
-                  parameters: (parameter_list
-                    (parameter
-                      type: (predefined_type)
-                      name: (identifier)))
-                  (block
-                    (return_statement
-                      (identifier))))))))
+              (anonymous_method_expression
```

Comment:
Why is it we're losing "static" as a (modifier) here but we're keeping static as a modifier on the lambda equivalent on line 1262?

Reply:
Gotcha, but what about static/unsafe in using_directive? I also (personally) think it's ok to hide modifiers, and just expose the string literals in the tree instead, but I don't have much of a preference.